### ISSUE TITLE
extra.task is logged poorly

### ISSUE THREAD

**Issue by OrChen17** [Intsights/sergeant#1]
using the provided logstash handler, extra.task is logged as follows:
```
extra.task: Task(kwargs={}, date=1590487633, run_count=0)
```
Instead of a desired dict

**Comment by OrChen17** [comment]
I would add to logstash_handler (line 84):
```
        for attribute_name, attribute_value in record.__dict__.items():
            if attribute_name not in self.logrecord_attributes:
                if dataclasses.is_dataclass(attribute_value):
                    attribute_value = dataclasses.asdict(
                        attribute_value,
                    )
                message['extra'][attribute_name] = attribute_value
```

**Comment by wavenator** [comment]
Thanks for the report!
fixed on version 0.12.2
